qutip/nonmarkov/bofin.py

```diff
@@ -1048,17 +1044,16 @@ def _configure_solver(self):
         RHSmat = RHSmat.tocsr()
 
         if self.isTimeDep:
-            solver_params = []
             h_identity_mat = sp.identity(nstates, format="csr")
             H_list = self.H_sys.to_list()
 
-            # Store each time dependent component
+            # store each time dependent component
+            solver_params = [[RHSmat]]
```

Comment:
@nwlambert I added the RHS matrix to the start of the solver_params to the time-dependent fermionic case because it wasn't there before. None of the tests broke and I added a dummy time-dependent test in an earlier commit. Is this the correct change to make?

Reply:
Yikes, yeah good spot.  Next time we chat we can sort out any missing tests to catch this kind of thing.